### PR TITLE
Cards for TTJets 1 lepton extension, using Madspin.

### PR DESCRIPTION
**Pull request by nstrobbe**
Adaptations of inclusive LO cards to give semileptonic decays of ttbar. Two sets of cards to circumvent madspin limitations.

### CODE REVIEW

**Comment by stosi** [comment]
Everything seems OK to me

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/tt0123j_1l_t_5f_ckm_LO_MLM/tt0123j_1l_t_5f_ckm_LO_MLM_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#to properly limit the number of concurrent processes for grid running
+set max_running_process 1
+#
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+
+# specify the decay for the final state particles
+decay t > w+ b, w+ > ell+ vl
+decay t~ > w- b~, w- > j j
```
Comment (bendavid) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/tt0123j_1l_t_5f_ckm_LO_MLM/tt0123j_1l_t_5f_ckm_LO_MLM_madspin_card.dat]: not 100% sure j will include the b's in this case.

Might want to explicitly define the jets (or define a "q" multiparticle here) to be sure.

**Comment by nstrobbe** [comment]
I defined a q multiparticle and updated the madspin cards

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/tt0123j_1l_tbar_5f_ckm_LO_MLM/tt0123j_1l_tbar_5f_ckm_LO_MLM_madspin_card.dat [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# set BW_cut 15                # cut on how far the particle can be off-shell
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+# 
+#to properly limit the number of concurrent processes for grid running
+set max_running_process 1
+#
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+define q = u u~	d d~ c c~ s s~ b b~
```
Comment (bendavid) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/tt0123j_1l_tbar_5f_ckm_LO_MLM/tt0123j_1l_tbar_5f_ckm_LO_MLM_madspin_card.dat]: This line has some weird spacing.  Are you sure it's ok?  (Generally being paranoid about this sort of thing causing silent unexpected results)

**Comment by nstrobbe** [comment]
ah, that's strange, I didn't see that in my editor. Should be fixed now.

**Pull request merged**